package.json

```diff
@@ -45,7 +45,7 @@
   "dependencies": {
     "async": "1.5.2",
     "chalk": "1.1.3",
-    "chokidar": "1.5.2",
+    "chokidar": "2.1.2",
```

Comment:
this is a breaking change, isnæt it?

Reply:
Have a test locally, because it will have a low vulnerability of "brances" problem, and I know it is a breaking change but it seems everything is right on my test locally, and let's see the final result on the CI :)

Reply:
And I'm not sure whether the Nodejs's version will updated to the stable ones such as 6,8,10? Maybe yours are too old :)

Reply:
It seems everything goes right now, pass all the unit tests @SimenB

Reply:
It's mostly breaking for windows users, as if they do `path.resolve(from, to)`, the return value from that will no longer work in watch mode (path separators must be forward slash in newer micromatches, which was the breaking change in chokidar). Not sure how to best handle it. Could just release a major, I guess. I was working on a big v2 rewrite, but then I switched jobs and haven't really used stylus for a few years

Reply:
OK, I'll roll back and then there's only one change left. Your problem I'll see it in the future if free. Thanks very much !

Reply:
Thanks @SimemB，now I've changed my submit for one fixture, for the low vulnerbility I'll see it later :)

Reply:
Thanks @SimenB，now I've changed my submit for one fixture, for the low vulnerbility I'll see it later :)